Makefile.mingw64.mk

```diff
@@ -154,8 +154,11 @@ copy_zest_files:
 	cp $(shell find $(ZEST_PATH)/deps/nanovg -type f | grep ttf$$) $(ZYN_FUSION_OUT)/font/
 	cp $(ZEST_PATH)/src/osc-bridge/schema/test.json $(ZYN_FUSION_OUT)/schema/
 
-	cp $(ZYNADDSUBFX_BUILD_DIR)/src/Plugin/ZynAddSubFX/vst/ZynAddSubFX.dll $(ZYN_FUSION_OUT)/
-	cp $(ZYNADDSUBFX_BUILD_DIR)/src/zynaddsubfx.exe $(ZYN_FUSION_OUT)/
+	cp	 -r $(ZYNADDSUBFX_BUILD_DIR)/build/bin/ZynAddSubFX.lv2 $(ZYN_FUSION_OUT)/ZynAddSubFX.lv2/
+	cp	 -r $(ZYNADDSUBFX_BUILD_DIR)/build/bin/ZynAddSubFX.vst3 $(ZYN_FUSION_OUT)/ZynAddSubFX.vst3/
+	cp	$(ZYNADDSUBFX_BUILD_DIR)/build/bin/ZynAddSubFX-vst2.dll $(ZYN_FUSION_OUT)/
```

Comment:
IIRC from some early development, a few hosts will get uppity if the name of the vst2 dll changes. Either my memory is wrong or we ought to make sure the vst2 lib doesn't change filenames.

Reply:
DPF CMake module's default VST2 output name is suffixed with `-vst2`. I kept this because I was afraid of confusion of DLL names.

Maybe I should rename them into `ZynAddSubFX.dll` (Windows) or `ZynAddSubFX.so` (Linux), as your description?

Reply:
I'm perfectly fine with them using the -vst2 relabel in the cmake/dpf level, as long as they're renamed in the ZYN_FUSION_OUT folder. Right now a supported use case is unzipping that folder and pointing a DAW at it.

Reply:
Many hosts recognize VST plugins by FOURCC, for example, REAPER.If there are mutiple plugin files with different names but the same FOURCC, then only one plugin is recognized.

I'm not sure how other hosts behave.

Reply:
I'd say let's just make sure the name is consistent to minimize the chances of any oddball hosts behaving strangely.

Reply:
OK, that will be easy: simply rename during `cp`.